imports/ui/case/case-messages.jsx

```diff
@@ -14,20 +14,24 @@ import { formatDayText } from '../../util/formatters'
 import { matchWidth } from '../../util/cloudinary-transformations'
 import { attachmentTextMatcher } from '../../util/matchers'
 import UserAvatar from '../components/user-avatar'
+import TextField from 'material-ui/TextField'
 import styles from './case.mss'
 import themes from '../components/user-themes.mss'
 import colors from '../../mui-theme/colors'
 import { Link } from 'react-router-dom'
 import {
   subheaderStyle,
   infoIconStyle,
-  attachmentButtonStyle,
+  attachmentButtonStyle2,
```

Comment:
`attachmentButtonStyle` is not used anywhere else, so by doing this it would be no longer referenced. Why not changing it instead of adding "2"?